plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -732,7 +686,10 @@ func getUsedPorts(pods ...*api.Pod) map[int]bool {
 	for _, pod := range pods {
 		for _, container := range pod.Spec.Containers {
 			for _, podPort := range container.Ports {
-				ports[podPort.HostPort] = true
+				// TODO: Aggregate it at the NodeInfo level.
+				if podPort.HostPort != 0 {
```

Comment:
I see. Can you add a comment "0 is explicitly ignored in PodFitsHostPorts, which is the only function that uses this value" ?